helmfile/values/fluentd-configmap.yaml.gotmpl

```diff
@@ -46,43 +46,126 @@ aggregator:
       <match kubernetes.var.log.containers.fluentd-**>
         @type null
       </match>
-      <match **>
-        @id output-s3
-        {{ if eq .Values.objectStorage.type "gcs" -}}
-        @type gcs
-
-        project {{ .Values.objectStorage.gcs.project }}
-        keyfile /etc/fluent/config.d/gcp_credentials.json
-        bucket {{ .Values.objectStorage.buckets.scFluentd }}
-        object_key_format %{path}%{time_slice}_%{index}.%{file_extension}
-        {{ else if eq .Values.objectStorage.type "s3" -}}
-        @type s3
-
-        aws_key_id "#{ENV['AWS_ACCESS_KEY_ID']}"
-        aws_sec_key "#{ENV['AWS_ACCESS_SECRET_KEY']}"
-        {{ if eq .Values.fluentd.forwarder.useRegionEndpoint true -}}
-        s3_endpoint {{ .Values.objectStorage.s3.regionEndpoint }}
-        {{ end -}}
-        force_path_style {{ .Values.objectStorage.s3.forcePathStyle }}
-        s3_region {{ .Values.objectStorage.s3.region }}
-        s3_bucket {{ .Values.objectStorage.buckets.scFluentd }}
-        {{- end }}
-
-        path logs/%Y%m%d/${tag}/
-
-        <buffer tag,time>
-          @type file
 
-          path /opt/bitnami/fluentd/logs/buffers/s3
-          overflow_action block
-          total_limit_size 9GB
+      <match kubeaudit.**>
+        @type copy
+        <store>
+          @type relabel
+          @label @AUDIT
+        </store>
+        <store>
+          @type relabel
+          @label @STORE
+        </store>
+      </match>
 
-          timekey 10m
-          timekey_wait 1m
-          timekey_use_utc true
-          chunk_limit_size 50MB
-        </buffer>
+      <match kubernetes.**>
+        @type copy
+        <store>
+          @type relabel
+          @label @AUDIT
+        </store>
+        <store>
+          @type relabel
+          @label @STORE
+        </store>
       </match>
+
+      <label @AUDIT>
+        # Set aside Harbor Core Audit events
+        <match kubernetes.var.log.containers.harbor-core-**>
+          @type rewrite_tag_filter
+          <rule>
+            key message
+            pattern /AuditLog/
+            tag audit.${tag}
+          </rule>
+        </match>
+
+        # Set aside OpenSearch Master Audit events
+        <match kubernetes.var.log.containers.opensearch-master-**>
```

Comment:
Is it only the opensearch master node that emits audit logs?

Reply:
I haven't yet checked.